iroha/src/lib.rs

```diff
@@ -60,7 +62,16 @@ impl Iroha {
             Arc::clone(&world_state_view),
             transactions_sender,
         );
-        let sumeragi = Sumeragi::new();
+        //TODO: get peers from blockchain
```

Comment:
As I remember, @takemiyamakoto assumed we will have additional JSON file with trusted peers.